configure.in

```diff
@@ -13,6 +13,7 @@ AC_HEADER_STDC
 PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.0.0])
 PKG_CHECK_MODULES(JACK, [jack >= 0.80.0])
 PKG_CHECK_MODULES(LIBLO, liblo >= 0.24, LO_FOUND="yes", LO_FOUND="no")
+AC_CHECK_LIB([m],[pow])
```

Comment:
gcc 5.4.0 on Mint 18 x86_64. john-d-murphy ran into the [same issue before] (https://github.com/john-d-murphy/timemachine/commit/9df5565935af697306bb664afa0085d829837d51#diff-3b3a6ec97232deb43dc14319a73872c1) 

Linking to the math library is necessary if you use math functions. If it happens to work on your system without `-lm`, maybe because the compiler is inlining all of them, it's only a fluke.